source/data-operations-manual/key-processes/index.html.md.erb

```diff
@@ -322,6 +322,24 @@ for each resource you want to retire you’ll need to to create a row in the csv
 #### 4. Review this process and documentation
 Everything else is done so it’s worth reviewing this process and seeing if there’s anyway we can improve it!
 
+### Process for removing entities by retiring its endpoint and resource
+
+#### 1. Get a list of entities to remove
+The list of entities should contain details like entity-number, reference, name and org.
+
+#### 2. Find the fact hash for entity you want to remove
+On Datasette, find the collection db
+Open fact table
+Filter the fact table with org name and entity-number
+Find the the fact hash for the entity you want to remove
+
+#### 3. Find the endpoint and resource given entity details
+Open fact_resource table
+Get the resource hash using the fact hash identified in step 2
+Use digital_land/resource_endpoint table to get the endpoint hash
```

Comment:
can't there be multiple resource hashes per entity number? E.g if we have two active endpoints providing data for the same entity, we would have facts from multiple resources when searching by org name and entity number? i'm not 100% sure

Reply:
I feel like there should be unique fact for an entity when filtered using organisation and entity-number which can then be mapped to resource/endpoint
here we are kind of backtracking so there can be multiple facts for a resource but there will be unique resource for a given fact (which we want to get)